docs/buildah-import.md

```diff
@@ -0,0 +1,72 @@
+% BUILDAH(1) Buildah User Manuals
+% Buildah Community
+% JUNE 2017
```

Comment:
Yes makes sense.  Lets get this merged and then open a new pull request for this.